llvmlite/tests/test_binding.py

```diff
@@ -1273,22 +1274,82 @@ def test_inlineasm(self):
 
 
 class TestObjectFile(BaseTest):
+
+    mod_asm = """
+        ;ModuleID = <string>
+        target triple = "{triple}"
+
+        declare i32 @sum(i32 %.1, i32 %.2)
+
+        define i32 @sum_twice(i32 %.1, i32 %.2) {{
+            %.3 = add i32 %.1, %.2
```

Comment:
Should make `sum_twice` reference `sum`.  The current tests passes without the `.add_object_file`.